lib/host.js

```diff
@@ -1,13 +1,12 @@
 'use strict'
 
-module.exports = function (attachment) {
-  let shogun = process.env.SHOGUN
+module.exports = function (addon) {
   let appHost = process.env.HEROKU_POSTGRESQL_HOST
-  if (attachment.plan.name.match(/dev|basic/)) {
+
+  if (addon.plan.name.match(/dev|basic/)) {
```

Comment:
For what it's worth, this could be `.endsWith('dev') || .endsWith('basic')` to minimize the chance it inadvertently catches some new unrelated plan in the future, but it's probably fine as is.